data_model/src/transaction.rs

```diff
@@ -1014,9 +950,73 @@ pub mod error {
 
 #[cfg(feature = "http")]
 mod http {
+    #[cfg(not(feature = "std"))]
+    use alloc::vec::vec;
+    #[cfg(feature = "std")]
+    use std::vec;
+
+    use iroha_version::declare_versioned_with_scale;
     use warp::{reply::Response, Reply};
 
-    impl Reply for super::VersionedPendingTransactions {
+    use super::*;
+
+    declare_versioned_with_scale!(VersionedPendingTransactions 1..2, Debug, Clone, FromVariant, IntoSchema);
```

Comment:
Should `PendingTransactions` be versioned at all? we use this type only in queries

Reply:
idk, better safe than sorry?

Reply:
yeah, let's leave it as is for now